src/main/java/com/softserve/service/CategoryService.java

```diff
@@ -0,0 +1,27 @@
+package com.softserve.service;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import com.softserve.model.Category;
+import com.softserve.repository.ICategoryRepository;
+
+@Service
+public class CategoryService {
+
+	private ICategoryRepository categoryRepository;
+	
+	public CategoryService(ICategoryRepository categoryRepository) {
```

Comment:
I would recommend using @Autowired (Inversion of Control / Injection) instead of constructors